hadoop-ozone/dist/src/main/compose/common/grafana/dashboards/Ozone - Ozone Manager RocksDB.json

```diff
@@ -0,0 +1,2876 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": {
+          "type": "grafana",
+          "uid": "-- Grafana --"
+        },
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "graphTooltip": 0,
+  "id": 11,
+  "links": [],
+  "panels": [
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "beegq3lpjhvcwf"
```

Comment:
We could consider skipping the inclusion of the datasource `uid` to simplify the import process for users.
Ref: https://github.com/grafana/grafana/issues/60769